sources/packages/backend/apps/api/src/route-controllers/overaward/models/overaward.dto.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  AssessmentTriggerType,
+  DisbursementOverawardOriginType,
+} from "@sims/sims-db";
+import {
+  AWARD_VALUE_CODE_LENGTH,
+  MAXIMUM_AWARD_VALUE,
+} from "../../../utilities";
+import { Length, Max, Min } from "class-validator";
+
+export class OverawardBalanceAPIOutDTO {
+  overawardBalanceValues: Record<string, number>;
+}
+
+export class OverawardAPIOutDTO {
+  dateAdded: Date;
+  overawardOrigin: DisbursementOverawardOriginType;
+  awardValueCode: string;
+  overawardValue: number;
+  addedByUserFirstName?: string;
+  addedByUserLastName?: string;
+  applicationNumber?: string;
+  assessmentTriggerType?: AssessmentTriggerType;
+}
+
+export class OverawardManualRecordAPIInDTO {
+  @Length(AWARD_VALUE_CODE_LENGTH)
```

Comment:
Should we validate min/max here? The awards are supposed to be always 4 characters.

Reply:
Yes, with that assumption only I just used `@length()` alone.

Reply:
I saw the changed code 😉